libs/media-player/src/lib/audio-player.tsx

```diff
@@ -25,11 +25,11 @@ export const AudioPlayer = ({ audioUrl, mimeType }: AudioPlayerProps) => {
     return (
         <StyledAudioPlayer controls>
             {isAudioMIMEType(mimeType) ? (
-                <source src={audioUrl} type={mimeType} />
+                <source key={audioUrl} src={audioUrl} type={mimeType} />
```

Comment:
Do we really want the key here? I suppose it's not the end of the world, but it feels a bit awkward.

Reply:
We used the `mimeType` as we did for video-player.